catalystwan/utils/pydantic_validators.py

```diff
@@ -1,11 +1,11 @@
 import ipaddress
 from typing import Any
 
-from pydantic.v1 import BaseModel, root_validator
+from pydantic import BaseModel, root_validator
 
 
 class ConvertBoolToStringModel(BaseModel):
-    @root_validator  # type: ignore
+    @root_validator(skip_on_failure=True)  # type: ignore
```

Comment:
`root_validator` is deprecated in Pydantic V2. Let's use:
https://docs.pydantic.dev/latest/concepts/validators/#model-validators
example:
https://github.com/CiscoDevNet/catalystwan/blob/main/catalystwan/models/policy/policy_definition.py#L631